extensions/amp-story-panning-media/0.1/amp-story-panning-media.js

```diff
@@ -32,55 +35,105 @@ export class AmpStoryPanningMedia extends AMP.BaseElement {
     /** @private {?Element} */
     this.element_ = element;
 
-    /** @private {?Element} */
-    this.image_ = null;
+    /** @private {?Element} The element that is transitioned. */
+    this.ampImgEl_ = dev().assertElement(
+      this.element_.querySelector('amp-img')
+    );
 
-    /** @private {?string} */
-    this.x_ = null;
+    /** @public {?string} Sent to siblings to update their position. */
+    this.x = this.element_.getAttribute('x') || '0%';
 
-    /** @private {?string} */
-    this.y_ = null;
+    /** @public {?string} Sent to siblings to update their position. */
+    this.y = this.element_.getAttribute('y') || '0%';
 
-    /** @private {?string} */
-    this.zoom_ = null;
+    /** @public {?string} Sent to siblings to update their position. */
+    this.zoom = this.element_.getAttribute('zoom') || '1';
+
+    /** @private {Array<Element>} */
+    this.siblings_ = [];
   }
 
   /** @override */
   buildCallback() {
-    this.x_ = this.element_.getAttribute('x') || '0%';
-    this.y_ = this.element_.getAttribute('y') || '0%';
-    this.zoom_ = this.element_.getAttribute('zoom') || '1';
+    this.getSiblings();
```

Comment:
Would make sense to make getSiblings async so you can also wait for it here in case it loads after the promise below (so you avoid the race condition)